### PR TITLE
New login flow has been implementd.

### PR DESCRIPTION
**Pull request by sarath-rp**

### CODE REVIEW

File: Example/ZCRMiOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -133,6 +133,7 @@
 				TargetAttributes = {
 					607FACE41AFB9204008FA782 = {
 						CreatedOnToolsVersion = 6.3.1;
+						DevelopmentTeam = BQ6UR7PPCQ;
```
Comment (BoopathyPSiva) [inline Example/ZCRMiOS.xcodeproj/project.pbxproj]: Remove DevelopmentTeam

File: ZCRMiOS/Classes/ZohoCRMSDK.swift [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	public static let shared = ZohoCRMSDK()
+	private var isVerticalCRM: Bool = false
+	private var zcrmLoginHandler: ZCRMLoginHandler = ZCRMLoginHandler.init()
+	private var zvcrmLoginHandler: ZVCRMLoginHandler = ZVCRMLoginHandler.init()
+	
+	private init() {
+		
+		if let file = Bundle.main.path(forResource : "AppConfiguration", ofType: "plist" ) {
+			
+			let appConfiguration: [String : Any]  = (NSDictionary( contentsOfFile : file ) as? [String : Any])!
```
Comment (BoopathyPSiva) [inline ZCRMiOS/Classes/ZohoCRMSDK.swift]: Avoid force unwrap.

File: ZCRMiOS/Classes/ZohoCRMSDK.swift [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+			let crmAppConfigs: CRMAppConfigUtil = CRMAppConfigUtil(appConfigDict: appConfiguration)
+			let appType = appConfiguration["Type"] as? String
+			crmAppConfigs.setAppType(type: appType!)
```
Comment (BoopathyPSiva) [inline ZCRMiOS/Classes/ZohoCRMSDK.swift]: Avoid force unwrap.

File: ZCRMiOS/Classes/ZohoCRMSDK.swift [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+public class ZohoCRMSDK {
+	
+	public static let shared = ZohoCRMSDK()
+	private var isVerticalCRM: Bool = false
+	private var zcrmLoginHandler: ZCRMLoginHandler = ZCRMLoginHandler.init()
+	private var zvcrmLoginHandler: ZVCRMLoginHandler = ZVCRMLoginHandler.init()
+	
+	private init() {
+		
+		if let file = Bundle.main.path(forResource : "AppConfiguration", ofType: "plist" ) {
```
Comment (BoopathyPSiva) [inline ZCRMiOS/Classes/ZohoCRMSDK.swift]: we need to throw an error if we can't read Appconfiguration.plist

**Review by vijayakrishnap — APPROVED** [review]

**Pull request merged**